pulsar-broker/src/test/java/org/apache/pulsar/broker/service/BrokerServiceAutoTopicCreationTest.java

```diff
@@ -587,4 +587,17 @@ public void testExtensibleLoadManagerImplInternalTopicAutoCreations()
 
     }
 
+    @Test
+    public void testAutoPartitionedTopicNameWithClusterName() throws Exception {
+        pulsar.getConfiguration().setAllowAutoTopicCreation(true);
+        pulsar.getConfiguration().setAllowAutoTopicCreationType(TopicType.PARTITIONED);
+        pulsar.getConfiguration().setDefaultNumPartitions(3);
+
+        final String topicString = "persistent://prop/ns-abc/testTopic/1";
+
+        Assert.assertThrows(PulsarClientException.NotFoundException.class,
```

Comment:
Please add a UT to cover the case when `allowAutoTopicCreationWithLegacyNamingScheme`==true.